### PR TITLE
Bump version to v0.20.0

### PR DESCRIPTION
**Pull request by mzr1996**
Please check it.

### CODE REVIEW

File: docs/en/install.md [hunk 1]
```diff
@@ -10,8 +10,9 @@ The compatible MMClassification and MMCV versions are as below. Please install t
 
 | MMClassification version |     MMCV version      |
 |:------------------------:|:---------------------:|
-| dev                      | mmcv>=1.3.16, <=1.5.0 |
-| 0.19.0 (master)          | mmcv>=1.3.16, <=1.5.0 |
+| dev                      | mmcv>=1.4.4, <=1.5.0  |
+| 0.20.0 (master)          | mmcv>=1.3.16, <=1.5.0 |
```
Comment (Ezra-Yu) [inline docs/en/install.md:14]: mmcv>=1.4.2

File: docs/en/changelog.md [hunk 2]
```diff
@@ -1,5 +1,43 @@
...
+### Bug Fixes
+
+- Fix potential unexcepted behaviors if `metric_options` is not specified in multi-label evaluation. ([#647](https://github.com/open-mmlab/mmclassification/pull/647))
+- Fix API changes in  `pytorch-grad-cam&gt;=1.3.7`. ([#656](https://github.com/open-mmlab/mmclassification/pull/656))
+- Fix bug which breaks `cal_train_time` in `analyze_logs.py`. ([#662](https://github.com/open-mmlab/mmclassification/pull/662))
+
+### Docs Update
+
+- Update README in configs according to OpenMMLab standard. ([#672](https://github.com/open-mmlab/mmclassification/pull/672))
+- Update install guide and README. ([#624](https://github.com/open-mmlab/mmclassification/pull/624))
```
Comment (Ezra-Yu) [inline docs/en/changelog.md:39]: installation

File: docs/en/install.md (hunk 1 above)
Comment (Ezra-Yu) [reply docs/en/install.md:14]: we can consider updating the `1.5.0` to `1.6.0` now.

Comment (mzr1996) [reply docs/en/install.md:14]: CPU training/inference is supported after 1.4.4, So we will change the dependency to 1.4.4 next version.
As for the max version, keep it in this version. May update after discussion.

Comment (Ezra-Yu) [reply docs/en/install.md:14]: get it.

**Review by Ezra-Yu — APPROVED** [review]
LGTM.

**Pull request merged**